src/layouts/NavbarMenuMobile.svelte

```diff
@@ -23,11 +23,18 @@
   function toggleNavOpened() {
     navOpened = !navOpened;
   }
+  let y: number;
 </script>
 
 <div class="container-lg flex h-16 w-full flex-wrap items-center justify-between lg:hidden">
   <a href="/" class="block">
-    <img src={Logo} class="h-8 lg:h-10" class:hidden={headings.length > 0} alt="MultiQC logo" />
+    <img
+      src={Logo}
+      class="h-8 opacity-0 transition-opacity ease-in lg:h-10"
+      class:opacity-100={location.pathname === "/" && y > 85}
```

Comment:
I think the logo is hidden on all other pages with this change? This might fix it:
```suggestion
      class="h-8 transition-opacity ease-in lg:h-10"
      class:opacity-0={location.pathname === "/" && y <= 85}
      class:opacity-100={location.pathname === "/" && y > 85}
```

Reply:
Going to be lazy and apply comment and test in preview..

Reply:
Ah, fix worked but now it flashes on page load.. 🙄

Reply:
Fixed in 8e322f5